watch_connectivity_garmin/README.md

```diff
@@ -73,6 +73,45 @@ void example() {
 
 ```
 
+As of this moment you will have to develop your own Garmin watch app to send and receive messages, but here's a little start
+```cpp
+public function onMessageReceived(message) as Void {
+  if (message != null && message.isValid()) {
+    // this is a good message to use show this received data (the message is a map)
+    var messageFromPhone = message.content();
+    // and update our view for this message data
+    WatchUi.requestUpdate();
+  }
+}
+
+public function onTap(event as ClickEvent) as Void {
+  var listener = new $.CommListener();
+  var app = $.getApp();
+  Communications.transmit({
+    "clickX" => event.getCoordinates()[0],
+    "clickY" => event.getCoordinates()[1],
+    "content" => "Hello from the watch!",
+  }, null, listener);
+}
+```
+
+## Releasing (Android release build)
+The app communicates with the Garmin classes via its namespace. So when released these can be minified
+making them uncontactable. To prevent this from happening you will need to include the exclusions in your
+`proguard-rules.pro` file as in the example:
```

Comment:
This doesn't need to be in the readme. The plugin can provide these rules to consumer applications.